Whiskey/Tasks/CreateFile.ps1

```diff
@@ -0,0 +1,93 @@
+function New-WhiskeyFile
+{
+    [Whiskey.Task("CreateFile")]
+    [CmdletBinding()]
+    param(
+        [Parameter()]
+        [Whiskey.Context]
+        $TaskContext,
+
+        [Parameter()]
+        [string]
+        $Path,
+
+        [Parameter()]
+        [string]
+        $Content,
+
+        [Parameter()]
+        [bool]
+        $Force
+
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+    if ( -not $Path )
+    {                                                         
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message $pathErrorMessage
+        return
+    }
+
+    if ( -not $TaskContext )
+    {
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message $pathExistsErrorMessage
+        return
+    }
+
+    $Path = Join-Path -Path $TaskContext.BuildRoot -ChildPath $Path

```

Comment:
Normally, we would use `Resolve-WhiskeyTaskPath` to convert the paths passed to a task to absolute paths, but the new pattern is to use a `ValidatePath` attribute on a `Path` parameter. I've created issues for that work. Once it is done, we can merge it in here.